### PR TITLE
Fix importing

### LINKED ISSUE #15: Doesn't Work
`const mashov = require('node-mashov')`
when this line executes the following error is thrown.
`SyntaxError: Cannot use import statement outside a module
`
Is this project still being maintained ?

### PR DESCRIPTION
**Pull request by ofekashery**
Close #15.
- Fix ``Cannot use import statement outside a module `` error
- Fix ``window is not defined`` error
- Web works
- Node.js works

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "src",
     "src/index.d.ts"
   ],
-  "main": "src/index.js",
+  "main": "dist/node-mashov.min.js",
```
Comment (yardnsm) [inline package.json:22]: I'm not sure if it's better to have the minfied version as the entry point. The ["Authoring Libraries" guide](https://webpack.js.org/guides/author-libraries/#final-steps) uses the non-minified one.

```suggestion
  "main": "dist/node-mashov.js",
```

File: webpack.config.babel.js [hunk 2]
```diff
@@ -4,30 +4,43 @@ import path from 'path';
 import pkg from './package.json';
 
 export default env => ({
+  target: 'node',
+  node: {
+    process: false,
+  },
+
```
Comment (yardnsm) [inline webpack.config.babel.js:11]: Why is this needed?

Comment (ofekashery) [reply webpack.config.babel.js:11]: It's to make ``axios`` work both on Node.js and web.
See https://github.com/axios/axios/issues/456#issuecomment-285287911.
